apps/glusterfs/health.go

```diff
@@ -0,0 +1,123 @@
+package glusterfs
+
+import (
+	"bytes"
+	"encoding/gob"
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/heketi/heketi/pkg/glusterfs/api"
+
+	"github.com/boltdb/bolt"
+	"github.com/lpabon/godbc"
+)
+
+var clusterHealth = "ClusterHealth"
+
+type dbchecker struct {
+	CurrentTime time.Time
+}
+
+func NewHealthz() *api.Healthz {
```

Comment:
This function seems to be in an odd location between the type definition and that type's functions. Let's move it so it doesn't disrupt the flow when reading the code as much.

Reply:
removed